includes/blocks.inc

```diff
@@ -422,7 +422,7 @@ function islandora_solr_simple_search_form($form, &$form_state) {
   $form['simple']["islandora_simple_search_query"] = array(
     '#size' => '15',
     '#type' => 'textfield',
-    '#title' => '',
+    '#title' => 'Search For:',
```

Comment:
I can definitely make that t() change and remove the colon.

On the `invisible` front, I would default it to visible and let others choose to hide it rather than the other way around, similar to the Advanced Search block to maintain predictability/consistent use... Speaking of which, maybe matching the label to be the same as on that block and therefore use 'Search term' instead of 'Search for'?